heat_pump_adoption_modelling/pipeline/MCS/load_mcs.py

```diff
@@ -0,0 +1,137 @@
+# File: heat_pump_adoption_modelling/pipeline/MCS/load_mcs.py
+"""Loading MCS HP records and relevant EPC records."""
+
+import pandas as pd
+import re
+
+from heat_pump_adoption_modelling import PROJECT_DIR
+
+max_cost = 5000000
+max_capacity = 100
+epc_address_fields = ["ADDRESS1", "POSTTOWN", "POSTCODE"]
+epc_characteristic_fields = [
+    "TOTAL_FLOOR_AREA",
+    "CONSTRUCTION_AGE_BAND",
+    "BUILT_FORM",
+    "PROPERTY_TYPE",
+    "HP_INSTALLED",
+]
+mcs_path = "inputs/MCS_data/heat pumps 2008 to end of sept 2021 - Copy.xlsx"
+epc_path = "outputs/EPC_data/preprocessed_data/Q2_2021/EPC_GB_preprocessed_and_deduplicated.csv"
+# TODO: put in config
+
+
+def load_domestic_hps():
+    """Loads domestic MCS HP installation data from file
```

Comment:
By convention, doc strings are not written in 3rd person singular. Write "Load" instead of "Loads".